app/graphql/mutations/create_user_standing.rb

```diff
@@ -0,0 +1,75 @@
+module Mutations
+  class CreateUserStanding < ApplicationQuery
+    include QueryAuthorizeSchoolAdmin
+    argument :student_id, ID, required: true
+    argument :reason, String, required: true
+    argument :standing_id, ID, required: true
+
+    description "Create a new standing log"
+
+    field :user_standing, Types::UserStandingType, null: true
+
+    def resolve(_params)
+      notify(
+        :success,
+        I18n.t("shared.notifications.done_exclamation"),
+        I18n.t("mutations.create_user_standing.success_notification")
+      )
+      { user_standing: create_user_standing }
+    end
+
+    private
+
+    def create_user_standing
+      UserStanding.transaction do
+        user_standing =
+          UserStanding.create!(
+            user_id: student.user.id,
+            reason: @params[:reason],
+            standing_id: @params[:standing_id],
+            creator: current_user
+          )
+
+        send_email
+
+        user_standing
+      end
+    end
+
+    def send_email
+      current_standing, previous_standing =
+        user_standings.first(2).map(&:standing)
+
+      previous_standing =
+        previous_standing ||
+          Standing.where(school_id: resource_school.id, default: true).first
```

Comment:
Can't we add this as a method in school?
```suggestion
          current_school.default_standing
```